src/api/common/schema-validator.js

```diff
@@ -12,7 +12,8 @@ const ValidationError = require('./errors').ValidationError;
 let SCHEMAS = {};
 
 function isValidAddress(address: string): boolean {
-  return core.UInt160.is_valid(address);
+  return typeof address === 'string' && address.length && address[0] === 'r' &&
```

Comment:
Flow seems to be complaining about:
```
>>> true && 0 && false
0
```

You might need to wrap the return in Boolean to shut it up?